elastica/joint.py

```diff
@@ -222,51 +228,68 @@ def __init__(self, k, nu, kt):
             Damping coefficient of the joint.
         kt: float
             Rotational stiffness coefficient of the joint.
+        nut: float = 0.
+            Rotational damping coefficient of the joint.
+        use_static_rotation: bool
+            If True, the initial rotation offset between the two systems is recorded
+            and enforced throughout the entire simulation. (default=True)
         """
         super().__init__(k, nu)
         # additional in-plane constraint through restoring torque
         # stiffness of the restoring constraint -- tuned empirically
         self.kt = kt
+        self.nut = nut
+
+        # if a static rotation offset between the two systems should not be enforced,
+        # we set the relative rotation to the identity matrix. Otherwise
+        if use_static_rotation:
+            self.static_rotation = None
+        else:
+            self.static_rotation = np.eye(3)
```

Comment:
```suggestion
        # if the current rotation between the 2 rods is not the rest state,
        # then we set the rest rotation to the identity matrix.
        if not is_rest_joint_state:
            self.rest_rotation_matrix = np.eye(3)
```

Reply:
As `rest_rotation_matrix` is now passed as a parameter to the constructor of RigidJoint, this code is not necessary anymore and was removed in https://github.com/GazzolaLab/PyElastica/pull/135/commits/cb84464402707586dbc511df29a4156d42a7bdc0